packages/ui/src/Button.tsx

```diff
@@ -178,7 +178,16 @@ export const Button = forwardRef<HTMLElement, ButtonProps>(
                   color={iconLeftColor}
                 />
               )}
-              <TruncatedText text={capitalize ? children.toUpperCase() : children} />
+              <TruncatedText
+                text={capitalize ? children.toUpperCase() : children}
+                /*
+                  If a button is disabled and text is long we don't want to show 2 popups.
+                  1. In case a button is disabled and no `disabledTooltipVisible` is enforced, we just hide truncated popup.
+                  2. In case a button is disabled and disabledTooltip is hidden with a false flag, there is a space to show
+                  the truncated popup -> setting it to undefined will leave the default behaviour.
+                */
+                tooltipVisible={disabled && disabledTooltipVisible !== false ? false : undefined}
```

Comment:
nit: a bit more readable
```suggestion
                tooltipVisible={disabled && disabledTooltipVisible ? false : undefined}
```

Reply:
@danad02 this needs to stay the old way. It's important for us to check for an explicit `false` value.  What is tricky here is that `disabledTooltipVisible` can be `undefined`, there is a test for it.

Reply:
What is tricky here is that `disabledTooltipVisible` can be `undefined`. There is a test for it.

Reply:
Oh, you're right :smile: